When no text is selected, Tools->Move Selected Text is enabled

**Describe the bug**

When no text is selected, Tools->Move Selected Text is enabled


**To Reproduce**
Steps to reproduce the behavior:
1. Ensure no text is selected - the cursor is blinking in the editor 
2. Go to the Tools menu. Select Move Selected Text
3. An error dialogue box appears


**Expected behavior**
The menu entry should be disabled when no text is selected, just as it is if you e.g. select a row of a table. Then the popup error would be irrelevant. 

If this isn't technically possible then the menu item selection should at least avoid showing an error.

**Environment (please complete the following information):**
 - Version 0.75.0 release
 - OS: windows 10
 - Language: English

**Additional context**

The popup box shows this stack trace:


```
This is zim 0.75.0
Platform: nt
Locale: en_GB cp1252
FS encoding: utf-8
Python: (3, 10, 7, 'final', 0)
PyGObject: (3, 42, 1)

======= Traceback =======
  File "zim/actions.py", line 159, in _on_activate
  File "zim/actions.py", line 130, in __call__
  File "zim/gui/pageview/__init__.py", line 7484, in move_text
  File "zim/gui/pageview/__init__.py", line 8626, in __init__
AssertionError: No Selection present
```

--- a/zim/gui/pageview/__init__.py
+++ b/zim/gui/pageview/__init__.py
@@ -6567,6 +6567,8 @@ class PageView(GSignalEmitterMixin, Gtk.VBox):
 			self.actiongroup.get_action('edit_object').set_sensitive(False)
 			self.actiongroup.get_action('remove_link').set_sensitive(False)
 
+		self.actiongroup.get_action('move_text').set_sensitive(buffer.get_has_selection())
+
 		# Emit signal if passing through a link
 		link = buffer.get_link_data(iter)
 		if link:

